models/UserGroupsModel.js

```diff
@@ -0,0 +1,38 @@
+import { Sequelize, DataTypes } from 'sequelize';
+import { USER_GROUPS } from '../constants/tables';
+import { sequelize } from '../loaders/sequelize';
+import Group from './GroupModel';
+import User from './UserModel';
+
+const UserGroups = sequelize.define(
+  USER_GROUPS,
+  {
+    id: {
+      type: DataTypes.UUID,
+      allowNull: false,
+      defaultValue: Sequelize.UUIDV4,
+      primaryKey: true,
+    },
+    groupId: {
+      type: DataTypes.UUID,
+      references: {
+        model: Group,
+        key: 'id',
+      },
+    },
+    userId: {
+      type: DataTypes.UUID,
+      references: {
+        model: User,
+        key: 'id',
+      },
+    },
+  },
+  {
+    timestamps: false,
+  },
+);
+User.belongsToMany(Group, { through: UserGroups, uniqueKey: 'id' });
+Group.belongsToMany(User, { through: UserGroups, uniqueKey: 'id' });
+
+export default UserGroups;
```

Comment:
This is just for your information only:  Sequelize internally creates the many-to-many dependent table inside.
eg: 

    
    User.belongsToMany(Group, { through: UserGroups, uniqueKey: 'id' });
    Group.belongsToMany(User, { through: UserGroups, uniqueKey: 'id' });
    
would have created UserGroups table inside automatically by mapping primary keys of both the dependent ones.
Though it is very okay to mention explicitly to avoid confusion and overriding with your custom name.

Reply:
Also, it opens a set of related methods for you. Check this doc:
https://sequelize.org/master/manual/assocs.html#special-methods-mixins-added-to-instances